<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GymForBattle</class>
 <widget class="QWidget" name="GymForBattle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="p1_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2_label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="hp_p1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="hp_p2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
